### PR TITLE
[aws] sso login

### PR DESCRIPTION
**Pull request by danilofuchs**
Implements the only missing suggestion for AWS SSO

https://awscli.amazonaws.com/v2/documentation/api/latest/reference/sso/index.html#cli-aws-sso

### CODE REVIEW

File: dev/aws/sso.ts [hunk 1]
```diff
@@ -202,6 +202,21 @@ const completionSpec: Fig.Spec = {
...
+      name: "login",
+      description:
+        "Refreshes SSO access tokens and logs in. Required setup using ``aws configure sso``",
+      options: [
+        {
+          name: "--profile",
+          description:
+            "The name of the profile to use.\n\nIf not set, will use the default profile",
+          args: {
+            name: "string",
```
Comment (fedeci) [inline dev/aws/sso.ts:215]:
```suggestion
            name: "name",
```

File: dev/aws/sso.ts [hunk 2]
```diff
@@ -202,6 +202,21 @@ const completionSpec: Fig.Spec = {
         },
       ],
     },
+    {
+      name: "login",
+      description:
+        "Refreshes SSO access tokens and logs in. Required setup using ``aws configure sso``",
```
Comment (QuiiBz) [inline dev/aws/sso.ts:208]: Looks like we have some extra `:
```suggestion
        "Refreshes SSO access tokens and logs in. Required setup using `aws configure sso`",
```

File: dev/aws/sso.ts (hunk 1 above)
Comment (QuiiBz) [reply dev/aws/sso.ts:215]: Agree with @fedeci

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks a lot for this contribution!

We only have two comments to resolve before getting this PR merged in.

File: dev/aws/sso.ts (hunk 2 above)
Comment (danilofuchs) [reply dev/aws/sso.ts:208]: I've used other AWS commands as template. It seems like all of them use double backticks

**Review by QuiiBz — APPROVED** [review]
LGTM, thanks a lot for this PR!

**Pull request merged**